### ISSUE TITLE
Bug - attached files with Xoops 2.5.10

### ISSUE THREAD

**Issue by aquasquale** [XoopsModules25x/newbb#122]
I have the impression that there is still a bug with the attached files (or else it is only at home, but I do not think so because I have tested on several versions of xoops).

It is possible to post them on the forum, but when I want to open them I get this message:

Quote:
This site is inaccessible The web page at http://mysite.com ... chid = 1616766010 & post_id = 2 may be temporarily inaccessible or may have been moved permanently to another address Web.
ERR_INVALID_RESPONSE

While on final version 5.0, the attached file downloads correctly.

But on the other hand, on version 5.0, I could only attach one file at a time.
When I send a second, it removes the first.

So to summarize:

- 5.00 final, only one attachment is accepted by mail. if we send a second, it removes the first.
- 5.01 Beta 4, we can send more than one, but we can't get them back.

config : PHP 7.3 / MySQL : 5.6 / Xoops 2.5.10

**Comment by cesagonchu** [comment]
I get bug with Xoops 2.5.11 too!
I tried to send a zip file as an attachment and it considers that it is an image


```
Warning: Failed fetching image size of /tmp/php18p7tb, skipping max dimension check.. in file /class/uploader.php line 534
--
Warning: Failed fetching image size of /tmp/php18p7tb, skipping max dimension check.. in file /class/uploader.php line 557
Notice: Undefined variable: postParentObject in file /modules/newbb/include/form.post.php line 85
```
___
My config:

Apache/2.4.18 (Ubuntu)
PHP version 7.4.9
MySQLi extension mysqlnd 7.4.9

**Comment by mambax7** [comment]
Couldn't replicate it - I've added two zip files and one image, and all of them are there:
http://demo.xoops.org/modules/newbb/viewtopic.php?topic_id=2&post_id=4#forumpost4

![image](https://user-images.githubusercontent.com/613686/128824223-03cf71f7-31ae-4394-a9fe-b699b9cc6737.png)


XOOPS Version | XOOPS 2.5.11-Beta2
PHP Version | 7.3.29

**Comment by cesagonchu** [comment]
Can you try with xswatch4 theme?

**Comment by mambax7** [comment]
Still there:
![image](https://user-images.githubusercontent.com/613686/128826417-29672495-8277-4e95-b9ed-837eda89f4b5.png)

**Comment by mambax7** [comment]
Fresh upload:
![image](https://user-images.githubusercontent.com/613686/128826804-ee102927-ed82-48e5-8043-60ebeb906354.png)

**Comment by cesagonchu** [comment]
Okay, it looks like it depends on the file size. I tried with a lighter file and it worked.
I limited the forums to 999 KB but it shows an error even though my file is only 436.55 KB.

Try with this file (see link below)? This is the one that gave me the problem.
https://www.frxoops.org/modules/newbb/viewtopic.php?post_id=242657#forumpost242657

**Comment by cesagonchu** [comment]
Ah! It seems that it is rather the upper case letters that he does not like!

**Comment by mambax7** [comment]
So is this issue now solved? If yes, can you close it?
